### ISSUE TITLE
macOS launchd plist

### ISSUE THREAD

**Issue by caesar** [stripydog/kplex#20]
Would you accept a pull request with a plist file enabling use of kplex as a daemon on macOS?

I have something I use myself but obviously will do some more testing to make sure it is robust before submitting anything, if you do want to include it in core.

**Comment by caesar** [comment]
Just a note that I have added a homebrew formula for kplex to [Pavel's "navutils" tap](https://github.com/nohal/homebrew-navutils), including a launchd plist. But if you want to include the plist in this repo instead, that would be another option.

**Comment by stripydog** [comment]
Yes (and hey, sorry it's taking me 3 years to work through my issues list).  Happy to include the plist file (submit a pull request) or I can link to Pavel's git repo (or if you have a blog page or website discussing kplex on mac I could link to that).  I confess to knowing little about launchd.  I note your plist file runs kplex in the foreground, but running it in the foreground means it complains to stderr rather than syslog and then I've been trying to work out where stderr gets sent if you don't explicitly send it somewhere.  A brief google suggests that fork/exec with parent then exiting (ie normal daemon process) is not OK for launchd.  Maybe I should add in a "don't fork but do log to syslog" option.  Any thoughts you have on this appreciated (or maybe we should start a discussion on the google group)
